### PR TITLE
Migrate data from self hosted InfluxDB to Aiven for InfluxDB

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
# What changed, and why it matters

Migrated and edited content of help article - Migrating data from a self-hosted InfluxDB to Aiven

### CODE REVIEW

File: .github/vale/dicts/aiven.dic [hunk 1]
```diff
@@ -85,6 +85,7 @@ hyperloglogs
 hypertable
 hypertables
 InfluxDB
+influx
```
Comment (ftisiot) [inline .github/vale/dicts/aiven.dic:88]: why is this needed?

File: docs/products/influxdb/howto/migrate-data-self-hosted-influxdb-aiven.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - You can do this via the `Aiven Console <https://console.aiven.io/>`_ by navigating to the **Databases** tab on your InfluxDB service or via the :doc:`Aiven CLI </docs/tools/cli>` and `REST API <https://api.aiven.io/doc/>`_. 
+    - Optionally, you can also change the retention policies for the destination database. For more information, see :doc:`InfluxDB retention policies <../concepts/influxdb-retention-policy>`. 
+
+.. note:: 
+    The ``avnadmin`` admin user does not have full superuser access, so it is necessary to pre-create the database before transferring the data. 
+
+1. **Import the data:** You can now push the exported data to the destination Aiven service using the ``influx -import`` command. You will need to specify the host, port, username, and password of the Aiven for InfluxDB service and the path to the exported data. The following is an example command: 
+::
+
+    $ influx -import -host influx-testuser-business-demo.aivencloud.com -port 12691 -username 'avnadmin' -password 'secret' -ssl -precision rfc3339 -compressed -path ./weather.influx.gz
```
Comment (ftisiot) [inline docs/products/influxdb/howto/migrate-data-self-hosted-influxdb-aiven.rst:47]: We might want to explain what the various parameters mean

**Review by ftisiot — CHANGES_REQUESTED** [review]
Reads good, I feel providing more explanation about the parameters of the export import would help

File: .github/vale/dicts/aiven.dic (hunk 1 above)
Comment (harshini-rangaswamy) [reply .github/vale/dicts/aiven.dic:88]: > why is this needed?

Vale issue. ``influx -import`` was not a recognized word.

Comment (ftisiot) [reply .github/vale/dicts/aiven.dic:88]: ok

File: docs/products/influxdb/howto/migrate-data-self-hosted-influxdb-aiven.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Create the data export file
+---------------------------
+
+To export the data from a self-hosted InfluxDB service, you will first need to run the ``influx_inspect export`` command. This command will create a dump file of your data.
+
+The following is an example command:
+::
+
+    $ influx_inspect export -datadir "/var/lib/influxdb/data" -waldir "/var/lib/influxdb/wal" -out "/scratch/weather.influx.gz" -database weather -compress
```
Comment (ftisiot) [inline docs/products/influxdb/howto/migrate-data-self-hosted-influxdb-aiven.rst:19]: We should remove the `$` from the command

File: docs/products/influxdb/howto/migrate-data-self-hosted-influxdb-aiven.rst (hunk 2 above)
Comment (ftisiot) [reply docs/products/influxdb/howto/migrate-data-self-hosted-influxdb-aiven.rst:47]: and remove the `$`

File: .github/vale/dicts/aiven.dic (hunk 1 above)
Comment (ftisiot) [reply .github/vale/dicts/aiven.dic:88]: I believe the `influx -import` should be seen as code, therefore the line shouldn't be needed

**Review by ftisiot — DISMISSED** [review]
Some more feedback

File: .github/vale/dicts/aiven.dic (hunk 1 above)
Comment (harshini-rangaswamy) [reply .github/vale/dicts/aiven.dic:88]: double-checked, it is in code style, and yet I received the vale error. Hence I added it previously. Maybe I can remove it and see what happens.

**Review by fingon — APPROVED** [review]
LGTM (Although I wonder if there's some way how we could offer this without using local files as source, e.g. from InfluxData SaaS)
